src/Test/StateMachine/Internal/Sequential.hs

```diff
@@ -255,7 +255,8 @@ checkSequentialInvariant
     monitor $ label s
     pre $ precondition m cmd
     resp <- run $ liftSem (sem m miref) cmd miref
+    let m' = transition m cmd resp
     liftProperty $
-      counterexample ("\nThe post-condition for `" ++ s ++ "' failed!\n\n" ++ show m) $
+      counterexample ("\nThe post-condition for `" ++ s ++ "' failed!\n\n" ++ show m') $
```

Comment:
I see what you mean with it being weird to print the transitioned one now, :-).

Hmm...